service/node.go

```diff
@@ -58,3 +311,30 @@ func (s *service) NodeGetCapabilities(
 		},
 	}, nil
 }
+
+func validateDevice(devicePath string) (string, error) {
```

Comment:
`gofsutil.ValidateDevice` is not usable here?

Reply:
Hi @codenrhoden,

It didn't exist when I wrote the above code. I added `ValidateDevice` to `gofsutil` later. I will update to the latest version of gofsutil and use its `ValidateDevice`. Good catch.